networkd/network_server.ml

```diff
@@ -844,6 +844,32 @@ module Bridge = struct
 				Sysfs.bridge_to_interfaces name
 		) ()
 
+	let get_physical_interfaces _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			match !backend_kind with
+			| Openvswitch ->
+				Ovs.get_real_bridge name
+				|> Ovs.bridge_to_interfaces
+				|> List.filter (Sysfs.is_physical)
+
+			| Bridge ->
+				let iface =
+					match List.filter (fun iface -> not (String.startswith "vif" iface)) (Sysfs.bridge_to_interfaces name) with
```

Comment:
Samples of bridges on bridge backend:
[root@xrtmia-06-07 brif]# ls /sys/class/net/xenbr2/brif/
eth2  vif1.0
[root@xrtmia-06-07 brif]# ls /sys/class/net/xapi1/brif/
bond0  vif1.1
[root@xrtmia-06-07 net]# ls /sys/class/net/xapi2/brif
bond0.3201  vif1.2
[root@xrtmia-06-07 net]# ls /sys/class/net/xapi0/brif
eth2.3201

I am not able to found any existing function which can collect vlan interfaces from bridge.
(Used `vif` to collect them).